drivers/wifi/nrf700x/CMakeLists.txt

```diff
@@ -162,37 +162,60 @@ if(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE)
     message(FATAL_ERROR "QSPI node not found in dts")
   endif()
 
-  dt_nodelabel(nrf70_fw_node NODELABEL "nrf70_fw_partition")
-  if(NOT DEFINED nrf70_fw_node)
-    message(FATAL_ERROR "nrf70_fw_partition node not found in dts")
-  endif()
-  dt_prop(nrf70_fw_offset PATH ${nrf70_fw_node} PROPERTY reg INDEX 0)
-  if(NOT DEFINED nrf70_fw_offset)
-    message(FATAL_ERROR "nrf70_fw_partition.reg not found in dts")
-  endif()
-  math(EXPR NRF70_PATCH_ADDR "${xip_addr} + ${nrf70_fw_offset}")
-
-  find_package(Python3)
-  # Using python module instead of intelhex command line tool to handle windows
-  execute_process(
-    COMMAND ${Python3_EXECUTABLE}
-      -c "import sys; import intelhex; intelhex.bin2hex(sys.argv[1], sys.argv[2], int(sys.argv[3]))"
-      ${NRF70_PATCH}
+  if(CONFIG_PARTITION_MANAGER_ENABLED)
```

Comment:
These commits need merging together, can't add code in one commit then completely change it in another

Reply:
Sure, but thougt it makes it easier to review, can combine.